core/arch/arm/kernel/unwind_private.h

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+#include <malloc.h>
+#include <stddef.h>
+#include <util.h>
+
+#ifdef CFG_UNWIND
```

Comment:
minor: `#ifdef CFG_UNWIND` could be removed.

Reply:
True, will do